src/qforte/ucc/spqe.py

```diff
@@ -439,16 +430,16 @@ def update_ansatz(self):
                     if(self._verbose):
                         print(f"  {Nmu_tup[1]:10}                  {np.real(Nmu_tup[0]):14}")
                     if(Nmu_tup[1] not in self._tops):
-                        self._tops.insert(0,Nmu_tup[1])
+                        self._tops.insert(0,self._excitation_dictionary[Nmu_tup[1]])
                         self._tamps.insert(0,0.0)
-                        self.add_from_basis_idx(Nmu_tup[1])
+                        self._nbody_counts[len(self._pool_obj[self._excitation_dictionary[Nmu_tup[1]]][1].terms()[0][1]) - 1] += 1
```

Comment:
What is `_nbody_counts`? It's not documented.

Reply:
_nbody_counts is a list that stores the numbers of singles, doubles, .... in the final ansatz. It is being printed near the end of an SPQE run. I will add a comment with the explanation,